src/lib.rs

```diff
@@ -274,6 +274,33 @@ impl Element {
         self.children.iter_mut().find(predicate)
     }
 
+    /// Transverse element using an xpath-like string: root/child/a
```

Comment:
Yup, I figured. No worries, though; push the doc update along with any others :)